javascript/javascript1/week1/family-pizza.js

```diff
@@ -0,0 +1,14 @@
+
+const pizzasPrice = 70;
+let amountOfPizzas = 3;
+let pizzaSize = 'Family Pizza';
+
+if (pizzaSize)
```

Comment:
Since `pizzaSize` is a string this isn't really checking for anything except that the string contains letters - which it usually will do.
Consider checking what the string says like this:
```javascript
if (pizzaSize === 'Family Pizza')
```